layouts/partials/sidebar-vault.html

```diff
@@ -0,0 +1,14 @@
+{{/* The "active" toggle here may delay rendering, so we only cache this side bar menu for bigger sites. */}}
+{{ $shouldCache := ge (len .Site.Pages) 2000 }}
+{{ if $shouldCache }}
+{{ $mid := printf "m-%s" (.RelPermalink | anchorize) }}
+<script>
+ $(function() {
+ $("#td-sidebar-menu #{{ $mid }}").toggleClass("active"); 
```

Comment:
minor nit: indenting is a bit off here.  Suggest four spaces (or a tab if preferred) on this line and next.

Reply:
Done. Thanks!